p2p/protocol/identify/id_delta.go

```diff
@@ -15,6 +16,8 @@ const IDDelta = "/p2p/id/delta/1.0.0"
 
 // deltaHandler handles incoming delta updates from peers.
 func (ids *IDService) deltaHandler(s network.Stream) {
+	s.SetReadDeadline(time.Now().Add(StreamReadTimeout))
```

Comment:
I'd prefer to explicitly ignore the error.

Reply:
This is done.